dotCMS/src/main/java/com/dotcms/util/SecurityUtils.java

```diff
@@ -85,18 +84,6 @@ public static String stripReferer ( HttpServletRequest request, String referer )
                     if ( !UtilMethods.isSet( foundHost ) ) {
                         foundHost = APILocator.getHostAPI().findByAlias( refererHost, systemUser, false );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "foundHost". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)